Formula/openvino.rb

```diff
@@ -0,0 +1,203 @@
+class Openvino < Formula
+  desc "Open Visual Inference And Optimization toolkit for AI inference"
+  homepage "https://docs.openvino.ai"
+  url "https://github.com/openvinotoolkit/openvino/archive/refs/tags/2022.3.0.tar.gz"
+  sha256 "b8dc1880d9ab71bd24aa5b2565724c12fb78172613e23ce312e0b98a6d8a0dd7"
+  license "Apache-2.0"
+  head "https://github.com/openvinotoolkit/openvino.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/^(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "protobuf" => :build
+  depends_on "python@3.11" => :build
+  depends_on "pugixml"
+  depends_on "tbb"
+
+  on_arm do
+    depends_on "scons" => :build
+
+    resource "openvino_contrib" do
+      url "https://github.com/openvinotoolkit/openvino_contrib/archive/refs/tags/2022.3.0.tar.gz"
+      sha256 "245e19b4a2c926aedb764fecbd02c5023deab56eba497e4a3c8194e760e26413"
+    end
+
+    resource "arm_compute" do
+      url "https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v22.11.tar.gz"
+      sha256 "e20a060d3c4f803889d96c2f0b865004ba3ef4e228299a44339ea1c1ba827c85"
+    end
+  end
+
+  resource "onednn_cpu" do
+    on_intel do
+      url "https://github.com/openvinotoolkit/oneDNN/archive/44de3c3698b687c26e487fc8f0213fa487e8fe2c.tar.gz"
+      sha256 "2c6aa7d787a947aa032224683f216ab38c168de4aed61d0554671774060a3615"
+    end
+  end
+
+  resource "ade" do
+    url "https://github.com/opencv/ade/archive/refs/tags/v0.1.1f.tar.gz"
+    sha256 "c316680efbb5dd3ac4e10bb8cea345cf26a6a25ebc22418f8f0b8ca931a550e9"
+  end
+
+  resource "ittapi" do
+    url "https://github.com/intel/ittapi/archive/refs/tags/v3.23.0.tar.gz"
+    sha256 "9af1231808c602c2f7a66924c8798b1741d3aa4b15f3874d82ca7a89b5dbb1b1"
+  end
+
+  resource "xbyak" do
+    url "https://github.com/herumi/xbyak/archive/refs/tags/v6.63.tar.gz"
+    sha256 "16c60f0682502624115c4dc9fec66782ae68ef32e469946f50cd169179ea92bb"
+  end
+
+  resource "onnx" do
+    url "https://github.com/onnx/onnx/archive/refs/tags/v1.12.0.tar.gz"
+    sha256 "052ad3d5dad358a33606e0fc89483f8150bb0655c99b12a43aa58b5b7f0cc507"
+  end
+
+  # OpenVINO mistakenly tries to find gflags, while it's actually not needed in current configuration
+  patch :DATA
```

Comment:
Has this been submitted upstream?

Reply:
yes, here https://github.com/openvinotoolkit/openvino/pull/14817

Reply:
I've reverted this patch for now.